website/src/colors.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line import/prefer-default-export
+export const light = {
```

Comment:
We don’t do theming now but having already a bit of a structure does not hurt.

Also, I’m still playing around with the colors choices and names, so it’s not final yet. I will go through them with the design team as well.